paper/paper.md

```diff
@@ -0,0 +1,321 @@
+We have plans to teach more workshops with instructors who did not develop the
+curriculum to gather even more feedback.
+
+![The curriculum development framework.](development-framework.png)
+
+# Curriculum
+
+Our curriculum is tailored to people with no prior coding experience who want to
+learn how to use programming to perform data analysis (Figure 2).
+We aim to teach our learners not only the basics of performing data analysis,
+but also to provide them with a framework to perform rigorous reproducible
+research that can easily be shared with others.
+
+![Curriculum overview.](curriculum-overview.png)
+
+## Learning Objectives
+
+The key learning objectives for our curriculum are:
+
+1. Create clear and informative data visualizations in R, starting with messy data.
+1. Perform version control using the Unix shell and git.
+1. Create reproducible reports using R markdown.
+1. Share code with others on GitHub.
+
+We believe these skills provide a solid foundation from which learners can teach
+themselves any additional coding skills that they will need in the future.
+
+## Course Content
+
+Our curriculum consists of modules that cover setup, data analysis and
+visualization, version control, and writing reports.
+We also provide a goal for the workshop: to create a fictitious report for the
+United Nations on the relationship between gross domestic product (GDP), life
+expectancy, and CO~2~ emissions.
+The modules are:
+
+0. Setup
+1. Welcome
+1. R for plotting (uses the tidyverse R package [@wickham_welcome_2019])
+1. The Unix shell
+1. Git and GitHub
+1. R for data analysis (uses the tidyverse R package [@wickham_welcome_2019])
+1. Writing reports in R markdown (uses the rmarkdown R package [@xie_r_2018])
+1. Group practice exercises
+1. Where to go from here
+
+Each lesson builds on the previous lessons.
+The Unix shell, git, and GitHub are introduced using the files generated in the
+R for plotting lesson.
+The lesson content for subsequent modules is then intermittently committed and
+pushed to GitHub.
+The written report combines all of the skills learned previously to write a
+report that one could share with the United Nations.
+Then, the learners have a chance to put everything they have learned into
+practice by working in small groups on practice problems that cover the course
+content.
+Finally, we end with a short module recapping what the workshop covered and how
+learners can continue to get help and learn more after the workshop ends.
+
+## Instructional Design
+<!-- teaching philosophy / pedagogy -->
+
+Our modules and instructor suggestions are developed in the style of [Software Carpentry](https://software-carpentry.org/):
+
+1. Each module contains learning objectives at the beginning of each lesson and
+a summary of key points.
+1. The five core modules (2 to 6) are designed to be taught via live coding of
+the content to learners.
+This is a central feature of Carpentries lessons, and we believe it is a great
```

Comment:
The Carpentries

Reply:
I think this one is actually okay per their style guide: https://carpentries.org/about/#referencing-the-carpentries